.swiftlint.yml

```diff
@@ -9,6 +9,7 @@ disabled_rules:
 - cyclomatic_complexity
 - multiple_closures_with_trailing_closure
 - todo
+- static_over_final_class
```

Comment:
why are we disabling this?

Reply:
Removed final class and this rule exemption 👍